Userland/Utilities/loopdevice-add.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2024, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/ArgsParser.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+#include <fcntl.h>
+#include <sys/ioctl.h>
+
+static ErrorOr<void> do_ioctl(int fd, int request, int inode_fd)
+{
+    int value = inode_fd;
+    TRY(Core::System::ioctl(fd, request, &value));
+    outln("{}", value);
+    return {};
+}
+
+ErrorOr<int> serenity_main(Main::Arguments arguments)
+{
+    TRY(Core::System::pledge("stdio rpath"));
+
+    StringView inode_path;
+
+    Core::ArgsParser args_parser;
+    args_parser.set_general_help("Create a loop device");
+    args_parser.add_positional_argument(inode_path, "File path", "path");
+    args_parser.parse(arguments);
+
+    int inode_fd = TRY(Core::System::open(inode_path, O_RDONLY));
+
+    int devctl_fd = TRY(Core::System::open("/dev/devctl"sv, O_RDONLY));
```

Comment:
Why do we need these utilties to add and remove a loop device? can't mount do this?

Reply:
Good question. Technically you could manually create loop devices and mount them. I guess there's no other use, besides that.